### DISCUSSION TITLE
Persist Data Feature Not Working on Windows 11

### DISCUSSION THREAD

**Discussion by ayub-kokabi** [wxt-dev/wxt#1113] [Q&A]
Read Issue #1023, Still Unresolved

According to the documentation, I've tried using absolute paths on Windows 11. However, the chrome-data directory still isn't being created.

```
import { defineConfig, defineRunnerConfig } from "wxt";
import { resolve } from 'node:path';

// See https://wxt.dev/api/config.html
export default defineConfig({
  extensionApi: "chrome",
  modules: ["@wxt-dev/module-react"],
  runner: {
    startUrls: ["https://info.cern.ch/"],
    chromiumArgs: [`--user-data-dir="${resolve("C:\Users\me\Desktop\New folder\AutoRTL\.wxt\chrome-data")}"`]
  },
});
```

I should mention that when I run the following command in CMD, it successfully creates the chrome-data folder as expected. However, I'm unsure how to achieve the same result using the chromiumArgs configuration. The command I use is:
`"C:\Program Files\Google\Chrome\Application\chrome.exe" --user-data-dir="C:\Users\me\Desktop\New folder\AutoRTL\.wxt\chrome-data"`

**Comment by ayub-kokabi** [comment]
No any idea?

**Comment by aklinker1** [comment]
https://github.com/wxt-dev/wxt/issues/1023#issuecomment-2452569111
